### PR TITLE
[Documentation] Customization Guide: Models - fix

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | yes
| New docs?    | no
| BC breaks?      | no
| Related tickets | partially #5275 
| License         | MIT

### CODE REVIEW

File: docs/customization/model.rst [hunk 1]
```diff
@@ -86,6 +86,8 @@ The file should be placed in ``AppBundle/Resources/config/doctrine/Country.orm.y
 
 3. Finally you'll need to override the model's class in the ``app/config/config.yml``.
 
+Under the ``sylius_*`` where * is the name of the bundle of the model you are customizing, in our case it will be the ``AddressingBundle`` -> ``sylius_addressing``.
```
Comment (dantleech) [inline docs/customization/model.rst]: i think `*` should be in double back-ticks

File: docs/customization/model.rst [hunk 2]
```diff
@@ -94,7 +96,21 @@ The file should be placed in ``AppBundle/Resources/config/doctrine/Country.orm.y
...
+
+.. code-block:: bash
+
+    $ php app/console doctrine:schema:update --force
+
+* via migrations:
+
+.. code-block:: bash
+
+    $ php app/console doctrine:migrations:migrat
```
Comment (dantleech) [inline docs/customization/model.rst]: migrat => migrate

File: docs/customization/model.rst [hunk 3]
```diff
@@ -94,7 +96,21 @@ The file should be placed in ``AppBundle/Resources/config/doctrine/Country.orm.y
...
+
+5. Additionally if you want to give the administrator an ability to add a ``flag`` to any of Countries,
```
Comment (dantleech) [inline docs/customization/model.rst]: `an => the`, also "flag" should be in double quotes not backticks (its not a code term)

File: docs/customization/model.rst (hunk 2 above)
Comment (pamil) [inline docs/customization/model.rst]: Shouldn't we generate them first?

File: docs/customization/model.rst (hunk 1 above)
Comment (pamil) [inline docs/customization/model.rst]: In fact bundle's name is `SyliusAddressingBundle`, not `AddressingBundle`

File: docs/customization/model.rst (hunk 2 above)
Comment (lchrusciel) [inline docs/customization/model.rst]: `doctrine:migrations:migrate` !== `doctrine:schema:update --force`

We should emphasize that we recommend a migration way, and present whole process:
```bash
$ doctrine:migrations:diff
$ doctrine:migrations:migrate
```

File: docs/customization/model.rst [hunk 4]
```diff
@@ -94,7 +96,21 @@ The file should be placed in ``AppBundle/Resources/config/doctrine/Country.orm.y
                 classes:
                     model: AppBundle\Entity\Country
 
-4. Additionally if you want to give the administrator an ability to add a ``flag`` to any of Countries,
+4. Update tha database:
+
+* via schema:
```
Comment (lchrusciel) [inline docs/customization/model.rst]: I found `via schema` misleading. It is only a key word of command. Direct database table alter?

File: docs/customization/model.rst [hunk 5]
```diff
@@ -94,7 +96,21 @@ The file should be placed in ``AppBundle/Resources/config/doctrine/Country.orm.y
                 classes:
                     model: AppBundle\Entity\Country
 
-4. Additionally if you want to give the administrator an ability to add a ``flag`` to any of Countries,
+4. Update tha database:
```
Comment (pamil) [inline docs/customization/model.rst]: the*

**Comment by lchrusciel** [comment]
It could be useful if would add a note with link to symfony2 documentation about doctrine and database migration. [This section](http://symfony.com/doc/current/book/doctrine.html#creating-the-database-tables-schema) especially.

File: docs/customization/model.rst (hunk 3 above)
Comment (TheMadeleine) [inline docs/customization/model.rst]: It is a code term. A variable on the model ;)

**Comment by pjedrzejewski** [comment]
Awesome, thanks Magda! 👍

**Comment by pjedrzejewski** [comment]
Definitely good idea! /cc @TheMadeleine We can add a relation to one of entities.

**Pull request merged**